docs/docs/guide/google-tag-manager-and-growthbook.mdx

```diff
@@ -0,0 +1,348 @@
+        if (value === "variation-1") {
+          document.querySelector("banner1").innerHTML = "Click here in the next 24 hours";
+        } else if (value === "variation-2") {
+          document.querySelector("banner1").innerHTML = "Click here before this offer expires";
+        }
+      });
+    }
+  })();
+</script>
+```
+
+Here, we are instrumenting two different tests. First is our button (`button-1`) which is modified by a 2-way AB test (control and one variant). It uses a boolean flag (`green-button`) to toggle a CSS class. Second is our sticky banner (`banner-1`) which is modified by a 3-way test. It uses a string flag (`my-string-feature`), with values representing different test variations, to alter the text of our banner.
+
+Notice that we need to call `gb.loadFeatures()` and wait for the return from the GrowthBook app before actually implementing our AB test’s render logic. This delay is responsible for some render flickering while switching from a control to a variant for our tests.
+
+## Strategy 2: Page speed optimized
+
+We can eliminate the `gb.loadFeatures()` call from the first strategy, thus reducing the render flickering, if we are comfortable implementing our own inline experiments and their render treatments. This approach, while performant, is less scalable because it requires more manual modification of GTM tags for each new experiment run or for changes in targeting, rollout, etc.
+
+In this scenario, our updated code snippet would look something like this:
+
+```html
+<script>
+  (function() {
+    // Wait for the SDK to load before starting GrowthBook
+    if (window.growthbook) {
+      startGrowthbook();
+    } else {
+      document.querySelector("#growthbook-sdk").addEventListener("load", startGrowthbook);
+    }
+
+    function startGrowthbook() {
+      if (!window.growthbook) return;
+      var gb = new growthbook.GrowthBook({
+        attributes: {
+          id: "u1234" // TODO: Read user/device id from a cookie/datalayer
+        },
+        trackingCallback: function(experiment, result) {
+          // TODO: track experiment impression
+        }
+      });
+
+      // 2-way inline test
+      var result1 = gb.run({
+        key: "button-experiment",
+        variations: ["control", "green-button"],
+        weights: [0.5, 0.5], // Traffic split between the variations
+        coverage: 1.0  // What percent of overall traffic to include (0.0 to 1.0)
+      });
+      if (result1.value === "green-button") {
+        document.querySelector("#button-1").classList.add("green");
+      }
+
+      // 3-way inline test
+      var result2 = gb.run({
+        key: "banner-experiment",
+        variations: ["A", "B", "C"],
+        weights: [0.5, 0.25, 0.25],
+        coverage: 1.0
+      });
+      if (result2.value === "variation-1") {
+        document.querySelector("banner1").innerHTML = "Click here in the next 24 hours";
+      } else if (res.value === "variation-2") {
```

Comment:
A couple typos here.
1. `res.value` should be `result2.value`
2. The values should be `B` and `C`, not `variation-1` and `variation-2`